src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VideoFanTakeover.java

```diff
@@ -1,19 +1,24 @@
 package com.wikia.webdriver.pageobjectsfactory.componentobject.ad;
 
+import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.WikiaWebDriver;
 import com.wikia.webdriver.common.core.elemnt.Wait;
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
 
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 
 public class VideoFanTakeover {
-  public static final double IMAGE_ASPECT_RATIO = 2.459;
-  public static final double VIDEO_ASPECT_RATIO = 1.769;
   private static final int VIDEO_LENGTH = 6000;
-  private static final By VIDEO_IFRAME_SELECTOR = By.cssSelector(".video-ima-container iframe");
-  private static By videoContainerSelector = By.cssSelector(".video-ima-container.hidden");
+  private static final String VIDEO_IFRAME_SELECTOR = "#%s .video-ima-container iframe";
```

Comment:
I don't know if we use this convention but it's up to you: there is a convention to use `_FORMAT` suffix when you have constants used as first parameter for `String.format()`, so in the examples here you'd have: `VIDEO_IFRAME_SELECTOR_FORMAT`, `MOBILE_VIDEO_SELECTOR_FORMAT` etc. It's up to you to use this convention or not.